lib/ui/view/dm_user/dm_user_view.dart

```diff
@@ -1,355 +1,335 @@
-                                        const CustomStatus(
-                                          isActive: true,
-                                          data: '5',
-                                        ),
-                                      ],
+                                  ),
+                                  const SizedBox(height: 20.0),
+                                  Row(
+                                    children: [
+                                      _onlineIndicator(0xFF00B87C),
+                                      const SizedBox(width: 16.0),
+                                      Text(model.receiver.username,
+                                          style: const TextStyle(
+                                              fontSize: 16.0,
+                                              fontWeight: FontWeight.w700)),
+                                      const SizedBox(width: 8.0),
+                                      Image.asset('assets/images/status.png',
+                                          width: 16.67, height: 16.0)
+                                    ],
+                                  ),
+                                  const SizedBox(height: 5),
+                                  Text(model.bio,
+                                      style: const TextStyle(
+                                          fontSize: 14.0,
+                                          fontWeight: FontWeight.w400,
+                                          color: Color(0xFF3A3A3A))),
+                                  const SizedBox(height: 15),
+                                  _startMessage(model.receiver.username)
+                                ],
+                              ),
+                            ),
+                            GroupedListView<Message, String>(
+                              shrinkWrap: true,
+                              elements: model.chatMessages,
+                              groupBy: (message) {
+                                return message.getRelativeTime();
+                              },
+                              groupSeparatorBuilder: (value) {
+                                return _groupSeparator(value);
+                              },
+                              itemBuilder: (context, message) {
+                                return MessageView(message);
+                              },
+                              groupComparator: (groupOne, groupTwo) =>
+                                  groupOne.compareTo(groupTwo),
+                              itemComparator: (itemOne, itemTwo) =>
+                                  itemOne.id.compareTo(itemTwo.id),
+                            ),
+                            const SizedBox(height: 40)
+                          ],
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+                Align(
```

Comment:
Is this the reply chat box from channels or threads?
Please create a file for it in the widgets directory